renderdoc/driver/gl/gl_pixelhistory.cpp

```diff
@@ -234,33 +370,146 @@ bool PixelHistorySetupResources(WrappedOpenGL *driver, GLPixelHistoryResources &
   // SPIR-V shaders are always generated as desktop GL 430, for ease
   rdcstr spirvSource = GenerateGLSLShader(GetEmbeddedResource(glsl_pixelhistory_primid_frag),
                                           ShaderType::GLSPIRV, 430);
+  rdcstr msCopySource = GenerateGLSLShader(GetEmbeddedResource(glsl_pixelhistory_mscopy_comp),
+                                           ShaderType::GLSPIRV, 430);
+  rdcstr msCopySourceDepth = GenerateGLSLShader(
+      GetEmbeddedResource(glsl_pixelhistory_mscopy_depth_comp), ShaderType::GLSPIRV, 430);
```

Comment:
It doesn't make any difference right now and probably won't ever, but in case this causes unexpected bugs in future these shaders should be compiled as `ShaderType::GLSL` rather than `ShaderType::GLSPIRV` since we're not expecting to use them on SPIR-V on GL here.